src/containers/DevSite.jsx

```diff
@@ -72,6 +74,7 @@ class Devsite extends Component {
       navOpen: false,
       postMeta: undefined,
       routeChanged: false,
+      hasCreds: true,
```

Comment:
Why is this default true.

Reply:
since hasCredentials was giving a boolean for `false` or `true`, if user didn't have the create .env the function would return `false`, now this is not needed so I'm setting this to false now since hasCredentials only sends something back if `true`.